sdk/python/feast/client.py

```diff
@@ -977,197 +601,128 @@ def ingest(
                     * csv
                     * json
 
+            project: Feast project to locate FeatureTable
+
             chunk_size (int):
                 Amount of rows to load and ingest at a time.
 
             max_workers (int):
                 Number of worker processes to use to encode values.
 
-            disable_progress_bar (bool):
-                Disable printing of progress statistics.
-
             timeout (int):
                 Timeout in seconds to wait for completion.
 
-        Returns:
-            str:
-                ingestion id for this dataset
-
         Examples:
             >>> from feast import Client
             >>>
             >>> client = Client(core_url="localhost:6565")
-            >>> fs_df = pd.DataFrame(
+            >>> ft_df = pd.DataFrame(
             >>>         {
             >>>            "datetime": [pd.datetime.now()],
             >>>            "driver": [1001],
             >>>            "rating": [4.3],
             >>>         }
             >>>     )
             >>> client.set_project("project1")
-            >>> client.ingest("driver", fs_df)
             >>>
-            >>> driver_fs = client.get_feature_set(name="driver", project="project1")
-            >>> client.ingest(driver_fs, fs_df)
+            >>> driver_ft = client.get_feature_table(name="driver")
```

Comment:
I think `get_feature_table("driver")` should also work.

Reply:
Yup, that works. I'll update the example.